traffic_ops/app/lib/API/Deliveryservice.pm

```diff
@@ -153,21 +158,26 @@ sub show {
 	my $current_user = $self->current_user()->{username};
 	my @data;
 
-	if ( !&is_privileged($self) ) {
+	my $tenant_utils = Utils::Tenant->new($self);
+	my $tenants_data = $tenant_utils->create_tenants_data_from_db();
+
+	if ( !&is_privileged($self) and !$tenant_utils->ignore_ds_users_table()) {
 
 		# check to see if deliveryservice is assigned to user, if not return forbidden
```

Comment:
I assume you refer to the code in the lines below the added code.
Done. 
Do you know by any chance if this functionality is covered by UT?